### DISCUSSION TITLE
人类是否会毁于人工智能 | /cn/2017/07/superintelligence/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1015] [Comments]
https://yihui.org/cn/2017/07/superintelligence/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
我绝对相信一部分人（比如决策层）肯定不会被铁箱子忽悠，我担心的是一个大群体被少数极端人士通过技术手段忽悠。

> ——原帖发布于 2018-02-06 03:07:32

**Reply by yihui** [reply]
哦哦，了然

> ——原帖发布于 2018-02-06 05:29:14

[a comment by an automated account is not shown]

**Comment by botsp** [comment]
站在2023年的尾巴上再次眺望，还是很恐慌的，哈哈
